schema/schema.sql

```diff
@@ -0,0 +1,48 @@
+CREATE TABLE author (
```

Comment:
- [ ] Kindly make sure that the schema file is working fine as now when I get an error when I try to run the code

Reply:
Solved!